### ISSUE TITLE
netbox_napalm_plugin-api

### ISSUE THREAD

**Issue by misho23-23** [netbox-community/netbox-napalm#5]
**NetBox version**
What version of NetBox are you currently running? 
```
Python version: 3.9.13
NetBox version: 3.5-beta1
``` 

**Describe the bug**
I have just upgraded NetBox to v3.5-beta1 with Netbox Napalm Plugin - https://netbox-community.github.io/netbox-napalm/
problem is associated with Napalm added tabs id Device: /dcim/devices//status OR (lldp-neighbors, config)
when I receive error message:
```
<class 'django.urls.exceptions.NoReverseMatch'>

'netbox_napalm_plugin-api' is not a registered namespace inside 'plugins-api'
``` 

**To Reproduce**
1. Go to 'Devices'
2. Click on 'Device_some'
3. Click on 'Status' OR 'lldp neighbors' OR 'Config'

**Screenshots**
![image](https://user-images.githubusercontent.com/130065618/232434620-024e642c-3512-4bd6-9752-f3b49df5f0d5.png)

Are you able to help me how to solve problem?
Thank you!

**Comment by misho23-23** [comment]
upgrade Python 3.10.11
downgrade Netbox v3.4.8

**Comment by arthanson** [comment]
@misho23-23 I think this should be fixed with a new published v0.1.1 release, there were some missing files in the v0.1.0 release.  Please pip install --upgrade the package and let me know if any issues.
